src/Migrations/Version20191126190235.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
Nepalikinėk šiukšlių. Ši migracija nieko nedaro

Reply:
Kodas veikia :heavy_plus_sign: buvo atskira pasakita (gal ir be namų darbų) apie `Doctrine` :heavy_plus_sign: vėlai atrašau (jau greitai final atsiskaitymas) – tai už namų darbus balo nemažinu – palieku tavo sąžinei, ar verta pasigilinti